src/Utils/LocalizeVector.js

```diff
@@ -0,0 +1,14 @@
+const from = undefined,
+      skipUpdate = true;
+
+var globalPoint = new PIXI.Point();
+
+function localizeVector(localDisplayObject, globalVector, localVector) {
+    localDisplayObject.getGlobalPosition(globalPoint);
+    globalPoint.x += globalVector.x;
+    globalPoint.y += globalVector.y;
+
+    return localDisplayObject.toLocal(globalPoint, from, localVector, skipUpdate);
```

Comment:
`getGlobalPosition` already updates the transform so no need to do it twice here.